torch/csrc/jit/runtime/register_prim_ops.cpp

```diff
@@ -1011,9 +1011,15 @@ static const std::vector<OperatorGeneratorArgs> opGenArgs{
         [](Stack& stack) {
           auto accumulate = pop(stack).toBool();
           auto values = pop(stack).toTensor();
-          auto indices = pop(stack).to<c10::List<c10::optional<at::Tensor>>>();
+          auto indices = pop(stack).to<c10::List<at::Tensor>>();
```

Comment:
Should we do the same thing on line 1001 and line 988 above? (`_index_put_impl_.hacked_twin` and `index.Tensor_hacked_twin`)